### ISSUE TITLE
When the interface returns a List structure and uses responseBodyAdvice to configure a unified return structure, the document output type is 'array'.

### ISSUE THREAD

**Issue by hellomsg** [TongchengOpenSource/smart-doc#641]
When the interface returns a List structure and uses responseBodyAdvice to configure a unified return structure, the document output type is 'array'.

## Your Environment

* Include as many relevant details about the environment where you experienced the bug.

* smart-doc version: 2.7.7
* plugin version (e.g., smart-doc-maven-plugin or smart-doc-gradle-plugin): 2.7.7
* build tool version (maven or gradle): 3.8.6
* jdk version: 1.8.0_381

## Expected Behavior

If you're describing a bug, tell us what should happen.
If you're suggesting a change/improvement, tell us how it should work.

## Current Behavior

If describing a bug, tell us what happens instead of the expected behavior.
If suggesting a change/improvement, explain the difference from current behavior.

In the JSON document generated by smart-doc:swagger, when the interface returns a List, `"type": "array"`. I have configured a unified return structure using responseBodyAdvice, expecting the type to be an object.

Configuration:
```json
"responseBodyAdvice": {
    "className": "com.common.api.CommonResult"
}
```

Unified return definition:
```java
public class CommonResult<T> {
    /**
     * Status code
     */
    private long code;
    /**
     * Prompt message
     */
    private String message;
    /**
     * Return data
     */
    private T data;
    /**
     * Is successful
     */
    private Boolean success;
}
```

Interface definition:
```java
/**
 * Get country/region list
 * @return
 */
@GetMapping
public List<Country> list() {
    return countryService.selectAll();
}
```

Generated swagger document:
```json
"/api/public/countries": {
    "get": {
        "summary": "Get country/region list",
        "deprecated": false,
        "description": "Get country/region list",
        "tags": [
            "Country/Region"
        ],
        "responses": {
            "200": {
                "schema": {
                    "type": "array",
                    "items": {
                        "$ref": "#/definitions/CommonResultListCountry_response"
                    }
                },
                "description": "OK"
            }
        },
        "operationId": "apipubliccountries",
        "parameters": []
    }
},
...
"CommonResultListCountry_response": {
    "properties": {
        "code": {
            "format": "int64",
            "description": "Status code",
            "type": "number",
            "example": "0"
        },
        "message": {
            "format": "string",
            "description": "Prompt message",
            "type": "string",
            "example": ""
        },
        "data": {
            "description": "Return data",
            "type": "array",
            "items": {
                "$ref": "#/definitions/Country_response"
            }
        },
        "success": {
            "format": "boolean",
            "description": "Is successful",
            "type": "boolean",
            "example": "true"
        }
    }
}
```

## Possible Solution 
Not obligatory, but suggest a fix/reason for the bug, or ideas on how to implement the addition or change.

## Steps to Reproduce 
Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce if relevant.

**Comment by shalousun** [comment]
@hellomsg If interested, you can refer to community co-construction, or help me register users, so that the community can better promote.
如果感兴趣，可以参考社区共建，也可以帮我登记下用户，方便社区更好的推广https://github.com/TongchengOpenSource/smart-doc/issues/12

**Comment by hellomsg** [comment]
https://github.com/TongchengOpenSource/smart-doc/issues/599 
This should be the same issue; let's wait for the 3.0.0 version to go live and then check if it has been fixed.
跟这个应该是一个问题，带3.0.0版上线再看看有没有修复

**Comment by hellomsg** [comment]
> @hellomsg If interested, you can refer to community co-construction, or help me register users, so that the community can better promote.
如果感兴趣，可以参考社区共建，也可以帮我登记下用户，方便社区更好的推广#12

@shalousun 3.0.0 
This should be the same issue; let's wait for the 3.0.0 version to go live and then check if it has been fixed.
发布到仓库了吗？只能搜到 com.ly.smart-doc 的。com.github.shalousun 还是 2.7.7

**Comment by shalousun** [comment]
> > @hellomsg If interested, you can refer to community co-construction, or help me register users, so that the community can better promote.
如果感兴趣，可以参考社区共建，也可以帮我登记下用户，方便社区更好的推广#12
> 
> @shalousun This should be the same issue; let's wait for the 3.0.0 version to go live and then check if it has been fixed.
3.0.0 发布到仓库了吗？只能搜到 com.ly.smart-doc 的。com.github.shalousun 还是 2.7.7

The community's group ID has changed.
社区groupId变了

**Comment by hellomsg** [comment]
> > > @hellomsg 如果感兴趣，可以参考社区共建，也可以帮我登记下用户，方便社区更好的推广#12
> > 
> > 
> > @shalousun 3.0.0 发布到仓库了吗？只能搜到 com.ly.smart-doc 的。com.github.shalousun 还是 2.7.7
> 
> 社区groupId变了

Cannot resolve plugin com.ly.smart-doc:smart-doc-maven-plugin:3.0.0

**Comment by hellomsg** [comment]
This bug has been fixed in version 3.0.0.
这个 bug 3.0.0 修复了。
